PD-56  Rewrite TrueCommand + TrueNAS SCALE SMB Clustering docs (#1814)
truenas/documentation: content/_includes/ClusterWarnings.md

> SMB clusters created in TrueNAS SCALE Bluefin are not and will not be available for cluster expansion. Beginning with TrueNAS SCALE Cobia, we will have a method to enable new volumes for SMB cluster expansion.

Reviewer: *do*n'*t use will
*
```suggestion
SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion. Beginning with TrueNAS SCALE Cobia, we plan to implement a method to enable new volumes for SMB cluster expansion.
```
